projects/sbb-esta/angular-public/file-selector/src/_file-selector.scss

```diff
@@ -11,18 +11,30 @@
 
 @mixin fileSelectorButton() {
   @include buttons;
+  @include secondaryButton;
 
-  &:not(.sbb-file-selector-disabled) {
-    @include secondaryButton;
-    @include arrowbuttons;
+  @if $sbbBusiness {
+    .sbb-svgsprite-icon {
+      display: none;
+    }
 
+    &.sbb-file-selector-disabled {
+      background-color: $sbbColorMilk;
+      color: $sbbColorGraphite;
```

Comment:
I think it should be $sbbColorIron (see digital.sbb.ch, invision app)

Reply:
No, because it is $sbbColorIron with a 50% opacity as in the screenshot above... which I interpreted as $sbbColorGraphite